virttest/utils_misc.py

```diff
@@ -2900,6 +2903,11 @@ def format_windows_disk(session, did, mountpoint=None, size=None,
     if size:
         size = int(float(normalize_data_size(size, order_magnitude="M")))
 
+    if labletype == PARTITION_TABLE_TYPE_GPT:
```

Comment:
function \"create_partition_table_windows(session, did, labeltype)\" in utils_disk can be used directly.
utils_disk.create_partition_table_windows()

Reply:
updated, thanks